cdc/owner.go

```diff
@@ -352,7 +352,12 @@ func (o *Owner) newChangeFeed(
 		cancel()
 	}()
 
-	err = sink.Initialize(ctx, sinkTableInfo)
+	err = primarySink.Initialize(ctx, sinkTableInfo)
+	if err != nil {
+		log.Error("error on running owner", zap.Error(err))
+	}
```

Comment:
Should we return err and pause the initialization of changefeed here?

Reply:
That was written by Zixiong Liu.
I just change the var from `sink` to `primarySink`,because I want to use the sink package then and the var `sink` can prevent it,so I change the name of the var `sink`.